justfile

```diff
@@ -2,4 +2,5 @@ _default:
   @just --list
 
 build:
-  whiskers notepad-plus-plus.tera
+  whiskers templates/npp-editor.tera
+  whiskers templates/npp-ui.tera
```

Comment:
I don't see the need for the `npp` prefix, think it's simpler to have it read `editor.tera` and `ui.tera`.